### PR TITLE
Updated scientific-computing-with-python description

### LINKED ISSUE #52663: Incorrect info in the new Python curriculum description
### Describe the Issue

The description of the new Scientific Computing with Python (Beta) curriculum contains info related to the previous certification.

### Affected Page

https://www.freecodecamp.org/learn/scientific-computing-with-python/

### Steps to Reproduce

See affected page

### Expected behavior

The description should be updated to not include networking, relational databases and data visualization.

### Screenshots

image

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by mhumayunsaeed**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #52663
Closes #53345

### CODE REVIEW

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -747,9 +747,8 @@
   "scientific-computing-with-python": {
     "title": "Scientific Computing with Python (Beta)",
     "intro": [
-      "Python is one of the most popular, flexible programming languages today. You can use it for everything from basic scripting to machine learning.",
-      "In the Scientific Computing with Python Certification, you'll learn Python fundamentals like variables, loops, conditionals, and functions. Then you'll quickly ramp up to complex data structures, networking, relational databases, and data visualization."
-    ],
+      "The Scientific Computing with Python (Beta) curriculum will equip you with the skills to analyze and manipulate data using Python, a powerful and versatile programming language. You'll learn how to handle large datasets, perform complex calculations, and visualize your findings using a variety of tools.", 
+      "This comprehensive course will guide you through the fundamentals of scientific computing, including data structures, algorithms, and essential libraries like NumPy, pandas, and Matplotlib."
```
Comment (zairahira) [inline client/i18n/locales/english/intro.json:751]: Currently, the certification is not teaching visualization or handling large data sets.
Same is the case with the mentioned libraries.

**Review by zairahira — CHANGES_REQUESTED** [review]

**Review by mhumayunsaeed — COMMENTED** [review]
Updated according to the requested changes.

**Comment by ieahleen** [comment]
Do you have issues with fixing this? Please reach out in the discord server.

**Comment by Humayun-glitch** [comment]
No, actually. I have been very busy with my Bachelors final year project. Will get to this by this weekend.

**Pull request closed without merging**